Error: Cannot find module 'lighthouse/report/report-generator'

Just after installing lh-ci under node v16.2.0 `npm install -g lighthouse-ci@1`, I got this error:

```
lighthouse-ci --help  
node:internal/modules/cjs/loader:944
  throw err;
  ^

Error: Cannot find module 'lighthouse/report/report-generator'
Require stack:
- /Users/tmolin/.nvm/versions/node/v16.2.0/lib/node_modules/lighthouse-ci/lib/lighthouse-reporter.js
- /Users/tmolin/.nvm/versions/node/v16.2.0/lib/node_modules/lighthouse-ci/bin/cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:941:15)
    at Function.Module._load (node:internal/modules/cjs/loader:774:27)
    at Module.require (node:internal/modules/cjs/loader:1013:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at Object.<anonymous> (/Users/tmolin/.nvm/versions/node/v16.2.0/lib/node_modules/lighthouse-ci/lib/lighthouse-reporter.js:13:25)
    at Module._compile (node:internal/modules/cjs/loader:1109:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1138:10)
    at Module.load (node:internal/modules/cjs/loader:989:32)
    at Function.Module._load (node:internal/modules/cjs/loader:829:14)
    at Module.require (node:internal/modules/cjs/loader:1013:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/tmolin/.nvm/versions/node/v16.2.0/lib/node_modules/lighthouse-ci/lib/lighthouse-reporter.js',
    '/Users/tmolin/.nvm/versions/node/v16.2.0/lib/node_modules/lighthouse-ci/bin/cli.js'
  ]
}
```

Downgrading to v1.12.0 is fine, but is there a workaround / fix 🤷

--- a/lib/lighthouse-reporter.js
+++ b/lib/lighthouse-reporter.js
@@ -10,7 +10,7 @@ const path = require('path');
 const { promisify } = require('util');
 const lighthouse = require('lighthouse');
 const chromeLauncher = require('chrome-launcher');
-const ReportGenerator = require('lighthouse/report/report-generator');
+const ReportGenerator = require('lighthouse/report/generator/report-generator');
 const chalk = require('chalk');
 const { createDefaultConfig, getOwnProps, convertToBudgetList, convertToResourceKey } = require('./helpers');
 

--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
   "type": "commonjs",
   "dependencies": {
     "chrome-launcher": "^0.14.0",
-    "lighthouse": "^8.1.0",
+    "lighthouse": "^8.4.0",
     "meow": "^9.0.0",
     "mkdirp": "^1.0.4",
     "ora": "^5.4.0",

